lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -184,3 +184,31 @@ end
     )
     @test isfile(joinpath(OUTPUT_DIR, "hybrid3d_cell.vtu"))
 end
+
+@testset "hybrid 3d sphere" begin
+    R = 6.37122e6
+
+    hdomain = Domains.SphereDomain(R)
+    hmesh = Meshes.EquiangularCubedSphere(hdomain, 4)
+    htopology = Topologies.Topology2D(hmesh)
+    quad = Spaces.Quadratures.GLL{5}()
+    hspace = Spaces.SpectralElementSpace2D(htopology, quad)
+
+
+    vdomain = Domains.IntervalDomain(
+        Geometry.ZPoint(0),
+        Geometry.ZPoint(20.0),
```

Comment:
```suggestion
        Geometry.ZPoint(R/3),
```

Reply:
There you go... if we used this z value, it'd be much more visible (tested -- images below)
![vtk-test-hybrid_sphere_point-output](https://user-images.githubusercontent.com/39932030/149432186-d777da3c-62ba-4331-a105-174c87c76efc.png)


![vtk-test-hybrid_sphere_point-output_wireframe](https://user-images.githubusercontent.com/39932030/149432173-e6eeaa52-f057-424f-817a-a00109ea2657.png)

Reply:
Ah, I just pushed a change which does a similar thing.

Reply:
That's fine as well! Thanks